sessionctx/variable/tidb_vars.go

```diff
@@ -1058,6 +1061,8 @@ var (
 	EnableFastReorg = atomic.NewBool(DefTiDBEnableFastReorg)
 	// DDLDiskQuota is the temporary variable for set disk quota for lightning
 	DDLDiskQuota = atomic.NewInt64(DefTiDBDDLDiskQuota)
+	// GeneralPlanCacheSize is a variable for general plan cache
+	GeneralPlanCacheSize = atomic.NewUint64(DefTiDBGeneralPlanCacheSize)
```

Comment:
Shouldn't put the `GeneralPlanCacheSize` here. The list only contains the `global-only` variables. See line 1019